packages/um7/package.py

```diff
@@ -13,6 +13,21 @@
 import os
 import os.path
 
+def _bld_cfg_path(opt_value, hg):
```

Comment:
Why is this a non-member function? You can make it a member function of the class `Um7` and then call it from within the class by doing `self._bld_cfg_path(...)`

Reply:
I had already tried using a member function and didn't like the `self` references. I will change it back.

Reply:
Done.